### ISSUE TITLE
Binding multiple values to ng-model using array

### ISSUE THREAD

**Issue by tushar579** [angular-ui/angular-ui-OLDREPO#571]
Hey All,

I am new to AngularJS. I need a bit of help of all of you.

I have written code to dynamically generate dropdownlists onclick of "Add more".

This is basically a form for an employee and an option for an employee to add more than one project in the form.

I am populating the values in the dropdown using ng-model and ng-repeat in the view.

For a scenario: Say I have 5 dropdowns right now, I want all the selected values of the dropdowns into an array and then send it to the server side code to store it in the DB.

But the problem is there is two-way binding of ng-model that is not allowing me to store the values in array. If I write the array, the dropdown stops populating.

Basically one to many relationship is implemented here with angualrjs.

Help appreciated.

**Comment by tushar579** [comment]
The sample code can be found at the following link:
https://groups.google.com/forum/#!topic/angular/2R7_pCC7E1o

**Comment by nateabele** [comment]
This repo has been discontinued. Please review the list of repositories in the organization, and reopen your issue against the relevant one. If you're looking for support, please try the #angularjs IRC channel on Freenode, or the [AngularJS](https://groups.google.com/forum/#!forum/angular) or [AngularUI](https://groups.google.com/forum/#!forum/angular-ui) mailing lists.
